client/src/lib/components/MonCompteProButton/MonCompteProButton.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  import franceConnect from "$lib/assets/france-connect.svg";
+</script>
+
+<div>
+  <button
+    class="fr-mb-1w"
+    aria-label="s'identifier avec Mon Compte Pro"
+    on:click
+  >
+    <img class="padlock" alt="" src={franceConnect} />
+
+    <p class=" text fr-pl-1w">
+      S'identifier avec <br />
+      <span class="fr-text--bold">Mon Compte Pro</span>
+    </p>
+  </button>
+
+  <a href=" http://mon-compte-pro.beta.gouv.fr/" target="_blank">
```

Comment:
Ici on a un [lien simple](https://gouvfr.atlassian.net/wiki/spaces/DB/pages/217284725/Liens+-+Links) (couleur bleue) de petite taille (cf maquette) donc il manque ces classes :

```suggestion
  <a class="fr-link fr-link--sm" href="http://mon-compte-pro.beta.gouv.fr/" target="_blank">
```